src/po-helpers.js

```diff
@@ -15,18 +31,15 @@ export function buildPotData(translations) {
     };
 
     const defaultContext = data.translations.context;
-    for (const trans of translations) {
-        if (!defaultContext[trans.msgid]) {
-            defaultContext[trans.msgid] = trans;
-            continue;
-        }
-        const oldTrans = defaultContext[trans.msgid];
 
-        // merge references
-        if (oldTrans.comments && oldTrans.comments.reference &&
-            trans.comments && trans.comments.reference &&
-            ! oldTrans.comments.reference.match(trans.comments.reference)) {
-            oldTrans.comments.reference = `${oldTrans.comments.reference}\n${trans.comments.reference}`;
+    if (isSortedByMsgid) {
+        for (const _ of Array(translations.length).keys()) {
+            const trans = translations.dequeue();
```

Comment:
looked closer at PriorityQueue class implementation, we can implement the desired behavior by cloning it's instance `priv.data` property. 
Example:
```js
const q = new PriorityQueue();
q.queue(2);
q.queue(1);
console.log(q.priv.data); // --> [1, 2]
```